pysper/search/filtercache.py

```diff
@@ -66,7 +66,7 @@ def __repr__(self):
             self.avg_evict_duration,
             self.byte_limit,
             self.item_limit,
-            self.evict_range,
+            int(self.evict_range),
```

Comment:
this can be float and in such case format will fail badly

Reply:
so I am not sure why you are casting this to int as we assume this is a float later?

Reply:
https://github.com/DataStax-Toolkit/sperf/blob/4b78e0eb1986c4e1afc0d9555fb92f4ca6bd020a/pysper/search/filtercache.py#L221

Reply:
I am casting it to int only for the `__repr__` function as it expects int with `{:d}`.

Reply:
ok so I read it again, yes I think it's good